.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Data request
+about: Suggest data to add to this platform
+
+---
+
+**Please describe the data requested**
+A clear and concise description of the data needed
+
+**How would this data be visualized?**
```

Comment:
Is this an optional field? Also, we should point them to the available chart types with the help text.

Reply:
@DavidLemayian can I point them to a country profile page because we don't have a list of available chart types?

Reply:
@phillipahereza By "point", I mean list them out.